README.md

```diff
@@ -91,21 +107,21 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 ./vendor/bin/psalm
 ```
 
-### Support the project
+## License
```

Comment:
The license section goes to the end, so it is in all repositories.

Reply:
It's not like that in recent package template.